reference/building-with-rollkit.md

```diff
@@ -30,17 +30,17 @@ There are currently 2 ways to build on Celestia:
 
 | network               | rollkit    | celestia-node | celestia-app |
 |-----------------------|------------|---------------|--------------|
-| local-celestia-devnet | v0.9.0     | v0.11.0-rc8   | v1.0.0-rc7   |
-| arabica               | v0.9.0     | v0.11.0-rc8   | v1.0.0-rc7   |
+| local-celestia-devnet | v0.10.1     | v0.11.0-rc8   | v1.0.0-rc7   |
+| arabica               | v0.10.1     | v0.11.0-rc8   | v1.0.0-rc7   |
 
 | rollkit/cosmos-sdk                          | rollkit/cometbft                   | rollkit    |
 |---------------------------------------------|------------------------------------|------------|
-| v0.46.13-rollkit-v0.9.0-no-fraud-proofs     | v0.0.0-20230524013049-75272ebaee38 | v0.9.0     |
-| v0.45.16-rollkit-v0.9.0-no-fraud-proofs     | v0.0.0-20230524013001-2968c8b8b121 | v0.9.0     |
+| v0.47.3-rollkit-v0.10.1-no-fraud-proofs     | v0.0.0-20230524013049-75272ebaee38 | v0.10.1     |
```

Comment:
should I remove the column for `rollkit/cometbft` here? @Manav-Aggarwal

Reply:
I think instead of removing it might be good to just update the corresponding version which is `0.37.2` for this release. cc @gupadhyaya